packages/cashc/src/grammar/CashScript.g4

```diff
@@ -120,7 +121,7 @@ numberLiteral
     ;
 
 typeName
-    : 'int' | 'bool' | 'string' | 'pubkey' | 'sig' | 'datasig' | Bytes
+    : 'int' | 'bool' | 'string' | 'date' | 'pubkey' | 'sig' | 'datasig' | Bytes
```

Comment:
Since we are converting the `date("x")` expression to an int I don't think we need an extra `date` type.

Reply:
For some reason when I was testing it wouldn't work if that wasn't there but it is fixed now.